legate/core/legion.py

```diff
@@ -5008,27 +5237,29 @@ def pack_buffer(self, buf):
     # Static member of this class for encoding dtypes
     _dtype_codes = {
         bool: legion.LEGION_TYPE_BOOL,
-        np.bool: legion.LEGION_TYPE_BOOL,
+        # np.bool: legion.LEGION_TYPE_BOOL,
```

Comment:
`np.bool` is just and alias for `bool` so I think including it in the dict is no-op (it is also deprecated as of np 1.20) Similar comments for `int`, `float` below.

Reply:
Ok. Can you actually make the change instead of commenting out those lines?

Reply:
:+1:  https://github.com/nv-legate/legate.core/pull/158/commits/00ff8701c9005dc9947b6875b5c6b25764c95e59